doc/whats_new/v0.22.rst

```diff
@@ -183,6 +183,14 @@ Changelog
   and non-contiguous arrays and makes a conversion instead of failing.
   :pr:`14458` by :user:`Guillaume Lemaitre <glemaitre>`.
 
+:mod:`sklearn.manifold`
+.......................
+
```

Comment:
I think we should also add SpectralEmbedding to Changed Models above as well, since this could be a breaking change for some user code. This also may affect SpectralClustering

Reply:
Added the 2 classes affected to the release notes.